src/arc_utilities/repeater.py

```diff
@@ -0,0 +1,27 @@
+import asyncio
+from threading import Timer
+from typing import Type
+
+import rospy
+
+
+class Repeater:
+
+    def __init__(self, topic_name: str, type: Type, period: float = 1.0, queue_size: int = 10):
+        """
+        @period publish the latest message every this many seconds
+        """
+        self.period = period
+        self.pub = rospy.Publisher(topic_name, type, queue_size=queue_size)
+        self.timer = Timer(self.period, self.run)
+        self.lock = asyncio.Lock()
+
+    def publish(self, msg):
```

Comment:
`def set_message` or `set_msg`.

`publish` does not actually publish. Since the repeater is not auto-run, this could be confusing.